Makefile

```diff
@@ -126,11 +136,13 @@ gen_key_cache: install_rs_build_toolchain
 		--features=$(TARGET_ARCH_FEATURE),shortint,internal-keycache -p tfhe
 
 .PHONY: build_core # Build core_crypto with and without experimental features
-build_core: install_rs_build_toolchain
+build_core: install_rs_build_toolchain install_rs_build_toolchain_nightly
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) build --release \
 		--features=$(TARGET_ARCH_FEATURE) -p tfhe
-	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) build --release \
-		--features=$(TARGET_ARCH_FEATURE),experimental -p tfhe
+	@if [[ "$(AVX512_SUPPORT)" == "ON" ]]; then \
+		RUSTFLAGS="$(RUSTFLAGS)" cargo +nightly build --release \
+			--features=$(TARGET_ARCH_FEATURE),experimental,$(AVX512_FEATURE) -p tfhe; \
+	fi
```

Comment:
if you look closely we build once with experimental, here if we don't have the avx512 we won't build with experimental anymore

Reply:
Copy paste mistake